### PR TITLE
[2.0.3-ea] Changelog update

### PR DESCRIPTION
**Pull request by aidanhahn**

### CODE REVIEW

File: charts/edge-stack/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ numbering uses [semantic versioning](http://semver.org).
 
 (no changes yet)
 
+## v7.1.8-ea
+
+- Update Edge Stack chart image to version v2.0.3-ea: [CHANGELOG](https://github.com/emissary-ingress/emissary/blob/master/CHANGELOG.md)
+
+# v7.1.3 - v7.1.7: Internal, not released
```
Comment (LukeShu) [inline charts/edge-stack/CHANGELOG.md:14]: This should surely have `##` rather than just `#`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -70,6 +70,13 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 (no changes yet)
 
+## [2.0.3-ea] September 16, 2021
+[2.0.3-ea]: https://github.com/datawire/edge-stack/compare/v2.0.2-ea...v2.0.3-ea
+
+### Ambassador Edge Stack
+
+(no changes yet)
```
Comment (LukeShu) [inline CHANGELOG.md:78]: Are we not including the Emissary changes here?

**Review by kflynn — DISMISSED** [review]
Looks good from here!

**Review by kflynn — APPROVED** [review]
Hit it!

**Pull request merged**